crates/sui-protocol-config/src/lib.rs

```diff
@@ -135,6 +136,10 @@ struct FeatureFlags {
     // compatibility error.
     #[serde(skip_serializing_if = "is_false")]
     missing_type_is_compatibility_error: bool,
+    // If true, then the scoring decision mechanism will not get disabled when we do have more than
+    // f low scoring authorities, but it will simply flag as low scoring only up to f authorities.
+    #[serde(skip_serializing_if = "is_false")]
+    scoring_decision_with_no_disable: bool,
```

Comment:
this flag name is a bit hard to understand. maybe `allow_bypass_scoring_decision`?

Reply:
Maybe `scoring_decision_with_validity_threshold_cut_off` ?